Sources/TestingMacros/Support/Additions/MacroExpansionContextAdditions.swift

```diff
@@ -13,18 +13,24 @@ import SwiftSyntaxBuilder
 import SwiftSyntaxMacros
 import SwiftDiagnostics
 
+#if !SWT_NO_LIBRARY_MACRO_PLUGINS
+private import Foundation
+private import SwiftParser
+#endif
```

Comment:
Curious about these `import` changes: what is Foundation being used for, and why are they guarded by `!SWT_NO_LIBRARY_MACRO_PLUGINS` even though that guard doesn't appear anywhere in this file?

Reply:
Leftover from an earlier experiment. Oops!

Reply:
Phew